Docs/Components_html/form.html

```diff
@@ -0,0 +1,180 @@
+                    <a href="./alert.html">Alert</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="./avatar.html">Avatar</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="./badge.html">Badge</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="./buttons.html">Button</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="./cards.html">Card</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="./grids.html">Grid</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="./image.html">Image</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="./inputs.html">Input</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="./lists.html">List</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="./modal.html">Modal</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="./navigation.html">Navigation</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="./ratings.html">Ratings</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="./toast.html">Toast</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="./form.html">Form</a>
+                </li>
+            </ul>
+        </aside>
+        <main class="main-section flex-column">
+            <section>
+                <h1 class="headings main-heading font-weight-4">Form Components</h1>
+                <p class="heading-about font-weight-3"> 
+                  Form components are used to save data form user.
+                </p>
+            </section>
+            <section class="component-section flex-column">
+                <div>
+                    <h2 class="heading sub-heading font-weight-4">Example of Sign Up form</h2>
+                    <p class="component-info">
+                        This is a sign up form, it is used when user wants to create account
+                    </p>
+                </div>
+
+                <div class="component-display-box font-weight-5 flex-row justify-center">
+                    <form class="form-box" action="#">
+                        <div class="form-head">
+                            <h4 class="sm-heading">Sign Up</h4>
+                        </div>
+                        <label class="form-label" for="first-name" aria-label="First Name">First Name
+                            <input class="input std-input" type="text" id="first-name" name="first-name" placeholder="Enter First Name">
+                        </label>
+                        
```

Comment:
i dont think label needs for since you have enclosed input inside label, rest all looks good

Reply:
Label is for accessibility purpose.